README.md

```diff
@@ -100,6 +100,8 @@ Django Commons packages.
 
 ## New Project Playbook
 
+Goal: Publish a new release of the project from django-commons repo to PyPI and Test PyPI, using the release workflow.
+
 Assuming the repository name is `repo-name`:
 
 ### Pre Transfer Steps
```

Comment:
This section probably should include defining the terraform repo code as per https://github.com/django-commons/membership/pull/82 once it's done.